### DISCUSSION TITLE
including modules when packing toga with beeware for OSX distro

### DISCUSSION THREAD

**Discussion by hoffenmerCode** [beeware/toga#1306] [Q&A]
I saw a similar question and answer on here but didn't get a clear answer from it so I'll ask afresh: 

Writing a toga app that import local modules and functions. 
They're in the same folder as the main file. 

When packing this for distro (dmg) for OSX I keep getting an error that the modules cannot be imported because they are missing or undefined. I know that when including pip-sourced modules (for lack of a better word) I just put that module name in the "required" field of the toml file. How would I include my local module to be packaged for distro? Guess I'd just include the file path so "required" : "/somePath/myfile.py" ?

**Comment by freakboy3742** [comment]
If your app contains multiple modules, you add the required directories to your `sources` declaration. You will need to have at least one `sources` reference that matches the name of your app - that becomes the app entry point, but if you need more than one top-level module, you can include as many as you want/need.
